nodes/sample_host.json

```diff
@@ -40,9 +40,10 @@
       "redirect_domain_names": ["<domain name>", "<domain name>", "<...>"],
       "ruby_version": "2.1.0",
       "ssl_info": {
-        "key": "<ssl key>",
-        "crt": "<ssl crt>"
-        },
+        "enabled": true,
+        "certificate": "The cert file, optional",
```

Comment:
Ah. In that case I'd communicate that:

  "certificate": "my_cert.crt (if not given, the app's name will be used, for ex: intercity_sample_app.crt)"
  "certificate_key": "my_cert.key"
Unfortunately one cannot add comments to a JSON file...


(After all, even when reading the code I was confused as how to use it. But that might just say about my Monday State Of Mind too :))